smrt/rtsolver/dort.py

```diff
@@ -145,13 +145,13 @@ def solve(self, snowpack, emmodels, sensor, atmosphere=None):
         """Solve the radiative transfer equation for a given snowpack, emmodels and sensor configuration.
 
         Args:
-            snowpack: Snowpack object, py:mod:`smrt.core.snowpack`.
-            emmodels: List of electromagnetic models object, py:mod:`smrt.emmodel`.
-            sensor: Sensor object, py:mod:`smrt.core.sensor`.
-            atmosphere: [Optional] Atmosphere object, py:mod:`smrt.atmosphere`.
+            snowpack: Snowpack object, :py:mod:`smrt.core.snowpack`.
```

Comment:
I think this will not be necessary when type hint will be added, but we can keep it now.